WHITESPACE overflows int on 32 bit compilation targets

```
github.com/spdx/gordf@v0.0.0-20201111095634-7098f93598fb/rdfloader/xmlreader/xmlreader.go:213:27: WHITESPACE (untyped int constant 4294977024) overflows int
```
Getting the following error when trying to compile for 32 bit targets.

--- a/rdfloader/xmlreader/readerUtils.go
+++ b/rdfloader/xmlreader/readerUtils.go
@@ -6,7 +6,7 @@ import (
 	"unicode"
 )
 
-const WHITESPACE = 1<<'\t' | 1<<'\n' | 1<<'\r' | 1<<' '
+const WHITESPACE uint64 = 1<<'\t' | 1<<'\n' | 1<<'\r' | 1<<' '
 
 type XMLReader struct {
 	fileReader *bufio.Reader

